assets/js/modules/analytics/components/dashboard/DashboardAllTrafficWidgetV2/index.js

```diff
@@ -20,31 +20,101 @@
  * WordPress dependencies
  */
 import { _x } from '@wordpress/i18n';
+import { useState, useEffect } from '@wordpress/element';
 
 /**
  * Internal dependencies
  */
 import Data from 'googlesitekit-data';
 import Widgets from 'googlesitekit-widgets';
-import { MODULES_ANALYTICS, FORM_ALL_TRAFFIC_WIDGET } from '../../../datastore/constants';
+import { FORM_ALL_TRAFFIC_WIDGET, DATE_RANGE_OFFSET, STORE_NAME } from '../../../datastore/constants';
```

Comment:
Let's use the new constants available and import this as `MODULES_ANALYTICS` instead, eg:

```suggestion
import { FORM_ALL_TRAFFIC_WIDGET, DATE_RANGE_OFFSET, MODULES_ANALYTICS } from '../../../datastore/constants';
```